### ISSUE TITLE
Column Width breaks when empty/null

### ISSUE THREAD

**Issue by KRN-8** [microsoft/vscode-hexeditor#337]
Sorry im sleepy & lazy + ENglish not my main language => Story time:
When you click the top left cog icon you get "Bytes per row" setting. If you change it, the changes reflect ASAP (which is nice, but). 
If you for example put "0" everything still works and the settings cog is still present, but if you want to write lets say "32" you would normally backspace/delete the current "16" (prolly default value) soo you could write anything else. But since changes happen immidiatley it sends "" making the internal value "hexeditor.columnWidth": "null" which breaks the whole thing and it gets stuck in loading, which makes the cog icon for changing this setting unavailable. 

Workaround:
Go into VSC Settings>Extensions>Hex editor>Collumn width, where you can reset this setting or edit the .json it self. 

TLDR:
you can set "hexeditor.columnWidth" setting to "null" by using hex editors GUI cog icon and emptying the "Bytes per row" field (probably missing a parse that puts 0 when empty string received)
![obrazek](https://user-images.githubusercontent.com/67783847/152877084-b956a166-81b1-4bbc-9e55-a736f0d0891f.png)

**Comment by lramos15** [comment]
Nice find it seems that max and min don't matter to the input and deleting it causes some weird behavior. This will be fixed in next release
